### PR TITLE
Version 17.19.0

### PR DESCRIPTION
**Pull request by chao-xian**
Includes:

* Remove expectation that sprockets is installed when used in a Rails app (PR #999)
* Fix tabs list overwrite on mobile (PR #1003)
* Set the origin when rendering the YouTube player (PR #1004)

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -161,7 +161,7 @@ GEM
     nokogiri (1.10.3)
       mini_portile2 (~> 2.4.0)
     null_logger (0.0.1)
-    oj (3.8.0)
+    oj (3.7.12)
```
Comment (andysellick) [inline Gemfile.lock]: Why the downgrade?

Comment (chao-xian) [reply Gemfile.lock]: I honestly don't know I'll check before I merge...

Comment (chao-xian) [reply Gemfile.lock]: Local git fubar so have reverted this

**Comment by chao-xian** [comment]
[Fix Smokey errors with youtube-nocookie](https://trello.com/c/YvdU9D39/33-fix-smokey-errors-with-youtube-nocookie)

**Pull request merged**